version/version.go

```diff
@@ -6,34 +6,16 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/refs"
 )
 
-// Version represents v2-compatible version.
+// Version represents version field in versioned structures of NeoFS.
 type Version refs.Version
 
-const sdkMjr, sdkMnr = 2, 11
+const sdkMjr, sdkMnr = 2, 12
 
-// NewFromV2 wraps v2 Version message to Version.
-//
-// Nil refs.Version converts to nil.
-func NewFromV2(v *refs.Version) *Version {
-	return (*Version)(v)
-}
-
-// New creates and initializes blank Version.
-//
-// Defaults:
-//  - major: 0;
-//  - minor: 0.
-func New() *Version {
-	return NewFromV2(new(refs.Version))
-}
-
-// Current returns Version instance that
-// initialized to current SDK revision number.
-func Current() *Version {
-	v := New()
+// Current returns Version instance that initialized to the
+// latest supported NeoFS API revision number in SDK.
+func Current() (v Version) {
```

Comment:
We use pointers everywhere in setters and getters, what is the reason for pointerless struct return here?

Reply:
I think @cthulhu-rider wants to remove all pointers from setters and getters (e.g. there is no pointers in client parameters). As soon as all packages will be documented and refactored this way, there will be no reference/derefences operations on SDK structures I suppose.

Reply:
i thought that we were changing only receivers, no?

but with the `Version` it is ok, I think

Reply:
@alexvanin ur right, instances generated using `Current` are read-only. Potential more flexible approach is
```
func WriteCurrent(v *Version)
```
but such flexibility does not seem to be in demand at the moment since it is useful for `Version` instance reuse only.